.github/workflows/website-sync-personio-jobs.yml

```diff
@@ -11,16 +11,20 @@ jobs:
   sync:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Fetch & Commit XML
+      - uses: actions/checkout@v3
+      - name: Fetch XML
         run: |
           curl https://wgtwo-jobs.personio.de/xml -o website/personio-jobs.xml
-          if [ ! -n "$(git status --porcelain)" ]; then
-            echo "No changes";
-            exit 0
-          fi
-          git config --global user.name 'GitHub Action'
-          git config --global user.email 'github-action@users.noreply.github.com'
-          git add website/personio-jobs.xml
-          git commit -m "Sync personio jobs xml"
-          git push
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          delete-branch: true
+          commit-message: "[autosync] Update personio job list"
+      - name: Enable Automerge on PR
+        if: steps.cpr.outputs.pull-request-operation == 'created'
+        uses: peter-evans/enable-pull-request-automerge@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We might need to create this GITHUB token?

Reply:
> Let's merge to test and see.

a-ok 👍

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Should be good. This token is auto generated